etc/unison/openlab-log-ha.prf.j2

```diff
@@ -0,0 +1,13 @@
+# Unison preferences file
+root = /srv/static/logs
+root = ssh://zuul@{{ logserver_opposite_ip }}//srv/static/logs
+batch = true
+owner = true
+group = true
+perms = -1
+repeat = 1800
+retry = 3
+sshargs = -i /var/lib/zuul/.ssh/id_rsa -C
+fastcheck = true
+log = true
+logfile = /var/log/unison/unison.log
```

Comment:
could be better to have comments on what these config means, it starts with # as line1

Reply:
OK, will do